### ISSUE TITLE
[Bug] Missing permission information for "Google Search Fixer" and "Video Background Play Fix" add-on

### ISSUE THREAD

**Issue by hwinnemoe** [mozilla-mobile/fenix#14671]
## Steps to reproduce
1. Open _Settings_, _Add-ons_, and tap on the +-Symbol next to the "Google Search Fixer" or "Video Background Play Fix" add-on

### Expected behavior
The user is informed that this add-on has permission to:
"Access your data for all websites" ("Google Search Fixer" add-on)

See also in the `manifest.json` of the "Google Search Fixer" WebExtension
```
"permissions": [
    "webRequest",
    "webRequestBlocking",
    "*://*/*"
  ]
```

### Actual behavior
The information about the required permissions is not displayed.

![add-on-no-permission-empty-s](https://user-images.githubusercontent.com/42788925/92101046-d6c45300-eddc-11ea-8bcf-8e1dbb669b9c.png)

### Device information

* Android device: Sony Xperia Z2, Android 6.0.1
* Fenix version: Nightly 200903 06:04 (Build #2015761713)
AC: 58.0.20200901190136, 478151899
GV: 82.0a1-20200831091558
AS: 61.0.13

**Comment by cadeyrn** [comment]
Depends on https://github.com/mozilla-mobile/android-components/issues/8323.

**Comment by hwinnemoe** [comment]
Leaving this open for visibility, @mozilladevs - feel free to close this if you think it's not needed.

**Comment by mcarare** [comment]
Will leave this open for ensuring this is tested by QA once the fix is implemented in AC.

**Comment by hwinnemoe** [comment]
This is now fixed in Nightly 200918 06:05 (Build #2015764593)
AC: 60.0.20200917130150, a1a5c582c
GV: 82.0a1-20200912220312
AS: 61.0.13.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build 200921`** using the following devices:
• Google Pixel 3a (Android 11)
• Huawei Mate 20 Lite (Android 10)
• OnePlus A3 (Android 6.0.1)
